OutOfSchool/OutOfSchool.Common/Models/CodeficatorAddressDto.cs

```diff
@@ -20,35 +23,42 @@ public class CodeficatorAddressDto
 
     public double Longitude { get; set; }
 
+    public int Order { get; set; }
+
     public string FullName
```

Comment:
![image](https://user-images.githubusercontent.com/11431506/182646756-f44d9241-2b29-46fa-b11e-5369d28c40e9.png)

Reply:
I know. This property needed at the front side.